packages/components/message-dialog/src/message-dialog.spec.ts

```diff
@@ -1,10 +1,10 @@
 import { expect } from '@open-wc/testing';
 import { type Dialog } from '@sl-design-system/dialog';
 import { sendKeys } from '@web/test-runner-commands';
-import { MessageDialog } from './message-dialog.js';
+import { html } from 'lit';
 import { spy } from 'sinon';
 import '../register.js';
-import { html } from 'lit';
+import { MessageDialog } from './message-dialog.js';
 
 describe('sl-message-dialog', () => {
```

Comment:
It looks like few tests are not working properly

<img width="840" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/6d7887ec-9b91-4185-a0f8-530d3592454d">

Reply:
Fixed